boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.dtsi

```diff
@@ -27,6 +27,7 @@
 		zephyr,console = &flexcomm4_lpuart4;
 		zephyr,shell-uart = &flexcomm4_lpuart4;
 		zephyr,canbus = &flexcan0;
+		zephyr,code-cpu1-partition = &slot1_partition;
```

Comment:
Here you are trying to use MCUBoot slot1 partition for Core1. 
But the current partition table was defined for MCUBoot - not for multi-core use-case.
Multi-core have to define own variant of partition table.

Reply:
I have created separate 200K partition for cpu1 image, would this be better option?
https://github.com/zephyrproject-rtos/zephyr/pull/81882/files#diff-62c4d7185a8523df62bb0a05951062b2770017f591408a07a0f5fb4777e20343R167

Reply:
> @butok , FYI. It has been common practice to place the code for the secondary core in the `slot1_partition`. LPC MCUs have used [zephyr,code-cpu1-partition](https://github.com/search?q=repo%3Azephyrproject-rtos%2Fzephyr%20zephyr%2Ccode-cpu1-partition&type=code) to do this, and RT11xx has used [nxp,m4-partition](https://github.com/search?q=repo%3Azephyrproject-rtos%2Fzephyr+m4-partition&type=code) to do the same. I assume it has been an easy option to re-use an existing flash partition that does not conflict with the primary core.

We have one common MCU-Boot specific partition setup that we try to use for all use cases.
The best solution would be something like the Nordic's Partition Manager (https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/scripts/partition_manager/partition_manager.html), but they don't plan to upstream it.